server/models/bookings.js

```diff
@@ -20,7 +20,16 @@ export default class Book {
 
   async decreaseNumberOfSeats(tripId, seatsAvailable, newSeats) {
     const newData = parseInt(seatsAvailable - newSeats);
+    const result = await db.updateBookings(tripId, newData);
+    return result;
+  }
+
+  async increaseNumberOfSeats(tripId, seatsAvailable, newSeats) {
```

Comment:
`decreaseNumberOfSeats` and `increaseNumberOfSeats` can be a single method try refactoring..